README.md

```diff
@@ -17,29 +17,36 @@ jobs:
       - uses: actions/checkout@v4
       - uses: scanner-inc/validate-detection-rules@v0.1.0
```

Comment:
since https://github.com/scanner-inc/validate-detection-rules/releases/tag/v0.1.0 doesn't have support for the validate_and_run_tests check_action, I think you'll have to bump this to v0.2.0 (or maybe v0.1.1 since it's backwards compatible, but semver pre v1 is kind of ambiguous IIRC).

And then reminder after you merge this to actually do the release (push the new tag).